tests/integration/test_crons.py

```diff
@@ -26,3 +29,57 @@ def test_monitors_with_processing(
         "status": "in_progress",
         "duration": 21.0,
     }
+
+
+def test_crons_endpoint_with_processing(
+    mini_sentry, relay_with_processing, monitors_consumer
+):
+    project_id = 42
+    options = {"processing": {}}
+    relay = relay_with_processing(options)
+    monitors_consumer = monitors_consumer()
+
+    mini_sentry.add_full_project_config(project_id)
+
+    monitor_slug = "my-monitor"
+    public_key = relay.get_dsn_public_key(project_id)
+    basic_auth = base64.b64encode((public_key + ":").encode("utf-8")).decode("utf-8")
+    relay.post(
```

Comment:
Just noticed, this should be `get` right @mitsuhiko ?

Reply:
The endpoint supports either.

Reply:
Ah cool. I'll expand on POST in a future PR.